.github/workflows/main.yml

```diff
@@ -100,6 +100,10 @@ jobs:
         with:
           arch: ${{ env.MODEL == '64' && 'x64' || 'x86' }}
 
+      - name: 'macOS 13: Switch to Xcode v14.1'
+        if: matrix.os == 'macos-13'
+        run: sudo xcode-select -switch /Applications/Xcode_14.1.app
```

Comment:
`/Applications/Xcode_14.3.1.app`?

Reply:
I guess that's the one which started with the first errors, so I thought better to start with the oldest. :)

Reply:
Based off what you posted, it was the switch from 14 to 15 that introduced it.

https://github.com/actions/runner-images/pull/9292/files#diff-49d5008075920b3466eff9e2bb3f4a933cca7733524c0153fd761cf9b6fc4fd4L173-R169